scripts/build

```diff
@@ -456,6 +461,32 @@ else
       echo "Executing (bootstrap): make $PKG_MAKE_OPTS_BOOTSTRAP" | tr -s " "
       make $PKG_MAKE_OPTS_BOOTSTRAP
       ;;
+
+    # python based build
+    "python:host")
+      echo "Executing (host): python setup.py build" | tr -s " "
+      export LDSHARED="$CC -shared $LDSHARED"
+      $TOOLCHAIN/bin/python setup.py build --build-base .$HOST_NAME
+      ;;
+    "python:target")
+      echo "Executing (target): python setup.py easy_install" | tr -s " "
+      mkdir -p .$TARGET_NAME
+      cp -r * .$TARGET_NAME
+      cd .$TARGET_NAME
+      export CFLAGS="$CFLAGS -I$SYSROOT_PREFIX/usr/include -L$SYSROOT_PREFIX/usr/lib"
+      export LDSHARED="$CC -shared $LDSHARED"
+      export PYTHONXCPREFIX="$SYSROOT_PREFIX/usr"
+      _prefix="$INSTALL/usr"
+      _pythonpath="$_prefix/lib/$PKG_PYTHON_VERSION/site-packages"
+      mkdir -p "$_pythonpath"
+      PYTHONPATH="$PYTHONPATH:$_pythonpath" \
+      $TOOLCHAIN/bin/python setup.py easy_install \
```

Comment:
easy_install will automatically download dependencies right? So this should be changed to "install"

Reply:
Do you really intend to keep checking dependencies manually or at runtime?

By the way, the current make_host/make_install_host automatically downloads, builds and installs dependencies too: the cffi:host of #2132 would build fine without pycparser:host. So you might want to correct that.

I see no added value in keeping the current toolchain, except for problematic Python packages.

Reply:
I noticed that too for host installs and I'm considering it a bug actually cause it's a side effect that can lead to weird error's.